### DISCUSSION TITLE
How to mark a notification as read with a shortcut outside the control center?

### DISCUSSION THREAD

**Discussion by mrwsl** [ErikReider/SwayNotificationCenter#251] [Q&A] [answered]
Maybe I'm blind but I can't figure out, how to close a notification with a keyboard shortcut outside the control centre. Is this possible?

**Comment by ErikReider** [comment, marked as answer]
There are two swaync-client commands that you can use: `--hide-latest` and `--close-latest`. Hope this helps :)

**Reply by mrwsl** [reply]
I'm indeed blind - thanks! `--close-all` is what I was looking for.
